javascript/stream_from_element.js

```diff
@@ -35,4 +35,6 @@ class StreamFromElement extends HTMLElement {
   }
 }
 
-window.customElements.define('stream-from', StreamFromElement)
+if (!window.customElements.get('stream-from')) {
+  window.customElements.define('stream-from', StreamFromElement)
+}
```

Comment:
Should we emit a warning if the `stream-from` element is already defined?

Reply:
Makes sense 👍🏼